package.json

```diff
@@ -61,11 +61,11 @@
     "typescript": "^5.4.5"
   },
   "dependencies": {
-    "@apidevtools/openapi-schemas": "^2.1.0",
     "@apidevtools/swagger-methods": "^3.0.2",
     "@jsdevtools/ono": "^7.1.3",
     "@readme/better-ajv-errors": "^1.6.0",
     "@readme/json-schema-ref-parser": "^1.2.0",
+    "@readme/openapi-schemas": "^3.0.1",
```

Comment:
if the newest version of `@readme/openapi-schemas` is `3.1.0`, does the dependency version also need to be updated?

Reply:
not sure i follow